backend/src/akvo/lumen/transformation/derive/js_engine.clj

```diff
@@ -13,8 +13,13 @@
                   {:value value
                    :type (type value)})))
 
-(defn- column-function [fun code]
-  (format "var %s = function(row) {  %s }" fun code))
```

Comment:
@kardan in relation to https://github.com/akvo/akvo-lumen/pull/1404#issuecomment-390584714
I've added this commit that removes the requirement for adding explicit `return` js word

no breaking API changes so we don't need to update Frontend code